linear_algebra/code/wg21_linear_algebra-config.cmake.in

```diff
@@ -3,15 +3,8 @@
 
 check_required_components(wg21_linear_algebra)
 
-if(NOT TARGET std::mdspan)
-    list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../../../cmake/")
-    include(${CMAKE_CURRENT_LIST_DIR}/../../../cmake/mdspanTargets.cmake)
-endif()
-
-if(NOT TARGET gtest)
-    list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../../../lib/cmake/GTest/")
-    include(${CMAKE_CURRENT_LIST_DIR}/../../../lib/cmake/GTest/GTestConfig.cmake)
-endif()
+include(CMakeFindDependencyMacro)
+find_dependency(mdspan REQUIRED)
```

Comment:
This is the correct way to find dependent packages: https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/